File: webservices/src/main/java/fieg/modulos/conciliacao/conciliaexternos/dto/ConciliacaoHitsDTO.java
package fieg.modulos.conciliacao.conciliaexternos.dto;


import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
public class ConciliacaoHitsDTO {

    private Integer sistema;
    private String entidade;
    private Boolean criadoCr5;
    private String motivoFalha;
    private LocalDateTime dataPagamento;
    private Integer numeroParcela;
    private BigDecimal valorPago;
    private String contratoProtheus;
    private Integer contrato;
    private String sacadoNome;
    private String sacadoCpfCnpj;
    private LocalDateTime dataInclusaoProtheus;
    private LocalDateTime dataAlteracaoProtheus;
}

File: web/src/models/DTOs/Hits/ConciliacaoHitsDTO.ts
export interface ConciliacaoHitsDTO {
  sistema: number;
  entidade: string;
  criadoCr5: boolean;
  motivoFalha: string;
  dataPagamento: number; // timestamp em milissegundos
  numeroParcela: number;
  valorPago: number;
  contratoProtheus: string;
  contrato: number;
  sacadoNome: string;
  sacadoCpfCnpj: string;
  dataInclusaoProtheus: number;
  dataAlteracaoProtheus: number | null;
}
